AUTHORS

```diff
@@ -1045,3 +1045,4 @@ zrm <trustiosity.zrm@gmail.com>
 蕭澧邦 <45505768+shou692199@users.noreply.github.com>
 谢乃闻 <sienaiwun@users.noreply.github.com>
 Нияз Гарифзянов <112617865+garrnizon@users.noreply.github.com>
+Jason C.H <ctrysbita@outlook.com>
```

Comment:
@ggerganov I am not sure what are the guidelines for modifying this file, I believe that until now it has been generated automatically from the git commits. Should this change be merged as is?

Reply:
Either way is fine - the `gen-authors.sh` script will re-generate the file.

I'll run it after this gets in.